frontend/packages/ceph-storage-plugin/src/utils/block-pool.tsx

```diff
@@ -235,7 +229,67 @@ export const FooterPrimaryActions = (t: TFunction) => ({
   UPDATE: t('ceph-storage-plugin~Save'),
 });
 
-export const isDefaultPool = (blockPoolConfig: StoragePoolKind): boolean =>
+export const isDefaultPool = (blockPoolConfig: K8sResourceKind): boolean =>
   !!blockPoolConfig?.metadata.ownerReferences?.find(
     (ownerReference) => ownerReference.name === OCS_INTERNAL_CR_NAME,
   );
+
+export const blockPoolColumnInfo = Object.freeze({
```

Comment:
I don't see the need for this abstraction. Why not use it directly inside the Header?

Reply:
ack

Reply:
I am using the same details in two places getColumn and getRow functions. So i kept this in some common place